examples/zkapps/12-cross-contract-calls/README.md

```diff
@@ -0,0 +1,26 @@
+# Mina zkApp: 12 Cross Contract Calls
```

Comment:
Hi @LuffySama-Dev  looks good so far!
For your README, use your own improved README "model" like https://github.com/o1-labs/docs2/blob/main/examples/zkapps/01-hello-world/README.md 

Thanks for all the docs love!

Reply:
Hi Barrie,

Sure, I will update it in next push.

Thank You 😃